src/map/index.js

```diff
@@ -17,9 +17,15 @@ const map = new mapboxgl.Map({
 const currentPos = document.createElement('div')
 currentPos.id = 'current-marker'
 const currentMarker = new mapboxgl.Marker(currentPos)
-  .setLngLat([locations[0].positions.lon, locations[0].positions.lat])
+  .setLngLat([
+    locations[0].positions.lon + 100,
```

Comment:
I'd probably stay away from changing actual coordinates of the marker just because when you zoom in far enough the marker will be displaced and wont show the actually position.